benchmarks/conftest.py

```diff
@@ -0,0 +1,152 @@
+from attr import dataclass
+import json
+import pytest
+
+from ariadne import QueryType, gql, make_executable_schema
+
+
+@dataclass
+class Person:
+    name: str
+    age: int
+
+
+@dataclass
+class User:
+    id: int
+    name: str
+    group: dict
+    avatar: list
+
+
+with open("benchmarks/simple.json") as f:
+    simple_data = json.load(f)
+
+with open("benchmarks/complex.json") as f:
+    complex_data = json.load(f)
+
+
+def person_data_dictionary():
+    person = simple_data[0]
+    return {"name": person["name"], "age": person["age"]}
+
+
+def user_data_dictionary():
+    user = complex_data[0]
+    return {"name": user["name"], "group": user["group"], "avatar": user["avatar"]}
+
+
+query = QueryType()
```

Comment:
Building on above ideas (two sets of data, one with dicts and other with objects) we could drop need for custom resolvers and executable schema fixtures.

Reply:
We are using only default resolvers now